packages/itwinui-react/src/core/Buttons/DropdownButton.test.tsx

```diff
@@ -41,11 +41,12 @@ it('should render in its most basic state', () => {
   expect(container.querySelector('.iui-button-icon')).toBeTruthy();
 });
 
-it('should update icon when menu opens or closes', async () => {
+it('should update when menu opens or closes', async () => {
   const { container } = renderComponent();
 
   const button = container.querySelector('.iui-button') as HTMLButtonElement;
   expect(button).toBeTruthy();
+  expect(button).not.toHaveAttribute('data-iui-has-popover', 'open');
```

Comment:
Would it be safer to have `expect(button).not.toHaveAttribute('data-iui-has-popover')`, i.e. confirm that the attribute doesn't exist regardless of the value? So, if `data-iui-has-popover: opened` is present instead of `data-iui-has-popover: open`, the test should fail?

Reply:
No, because it would also fail unnecessarily if we do something like `data-iui-has-popover="closed"` in the future.

Reply:
Ohh, I see. Yes, then as-is is good.

nit/optional: I was just suggesting this to confirm that `data-iui-has-popover` is not in the DOM, because the current check will not catch that. Also, maybe fixing a few tests may not be a big issue if we add `data-iui-has-popover="closed"` in the future.

Reply:
Leaving it as-is because checking whether `data-iui-has-popover` is removed from the DOM is not important. The CSS only cares about the exact `="open"` value, so it will not match even if a different value is used.